developer_tests/library/mkmf.template.gfortran

```diff
@@ -0,0 +1,147 @@
+# Template for GNU gfortran on Linux or Mac OSX
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+#
+# DART $Id$
+
+# typical use with mkmf
```

Comment:
Do we need to have all of this additional text in the comments from here (line 9) until line 131? In the newer mkmf.templates we've released (mkmf.template.quickscat.gfortran for example) we don't include a lot of this info. Just 114-131.

Reply:
no I don't think so, (just took it out). I think it is better to keep it simple.